airflow/providers/amazon/aws/operators/redshift_cluster.py

```diff
@@ -498,22 +502,38 @@ def __init__(
         wait_for_completion: bool = True,
         aws_conn_id: str = "aws_default",
         poll_interval: float = 30.0,
+        retry: bool = False,
+        retry_attempts: int = 10,
```

Comment:
I wonder if we should just have one `tries: int = 1`; having two separate arguments for this feels like a potential tripwire.

Reply:
That's a good idea. The only downside of it is you dont provide a default value in case you want to retry. If you want to retry, you have to come up with a number of retries which can be not very intuitive for users

Reply:
I think we need some standardization (see https://github.com/apache/airflow/pull/27276#discussion_r1021574968 )
I think it's better to decide on some standardization and then enforce it across. It's less productive when we discuss this per operator (and also may eventually have different names per operator)

Reply:
I am not sure it is the same use case though. In [Syed's PR](https://github.com/apache/airflow/pull/27276), the retry logic handles a bug in `boto3`. Here it handles a valid use case: removing a Redshift cluster that has a running operation. That's why here, in my opinion, we need a flag to whether activate the retry. Some users might want the operator to fail in such scenario, some might want to retry. Regarding the naming, I am happy to rename the parameters to `attempts` and `attempt_interval` to make it more consistent across the operators

Reply:
Unless I'm missing something the goal of these parameters from user perspective is to retry without counting this as Airflow retries so I do find this to be the same case (regardless of the reason that this functionality was needed to begin with)
WDYT?

Reply:
You are correct but the way I see it is one is a bug fix (through a hack) and one is a feature

Reply:
Should be good now @eladkal